README.md

```diff
@@ -3,23 +3,20 @@
 
 # Serverless Workflow Specification - Java SDK
 
-Provides the Java API/SPI and Model Validation for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
+Provides the Java API for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
 
 With the SDK you can:
 
-* Parse workflow JSON and YAML definitions
-* Programmatically build workflow definitions
-* Validate workflow definitions (both schema and workflow integrity validation)
-* Generate workflow diagram (SVG)
-* Set of utilities to help runtimes interpret the Serverless Workflow object model
+* Read workflow JSON and YAML definitions
+* Write workflow in JSON and YAML format. 
```

Comment:
For programatically building, I opened issue https://github.com/serverlessworkflow/sdk-java/issues/368
I want to emphasize we have writing utilities now, rather than generic utilities which I do not think will be needed with the new DSL.